aci/resource_aci_vzsubj.go

```diff
@@ -179,10 +218,72 @@ func setContractSubjectAttributes(vzSubj *models.ContractSubject, d *schema.Reso
 	d.Set("prio", vzSubjMap["prio"])
 	d.Set("prov_match_t", vzSubjMap["provMatchT"])
 	d.Set("rev_flt_ports", vzSubjMap["revFltPorts"])
+	d.Set("apply_both_directions", vzSubjMap["applyBothDirections"])
```

Comment:
do you actually get it from the API?

Reply:
no we don't get apply_both_directions from the API but if I set the parameter in setContractSubjectAttributes then I don't have to do it everytime i need to set parameters.